config/dynamic-scaling-configuration.yaml

```diff
@@ -3,10 +3,31 @@
 #The following properties must be defined for each instance types compute node configuration:
 #   - max_compute_nodes: the maximum number of nodes needed to create an appropriate machine pool with the configuration needed by the autoscaler
 
+# For each supported cloud provider, the following properties must be provided for the machine type
+#   - cluster_wide_workload_machine_type: the instance type size to handle cluster wide workloads e.g ingress controllers, observability operators, etc 
+#   - kafka_workload_machine_type: the instance type size to handle the Kafka instance workload e.g brokers, kafka-admin API, etc
+
 ---
-developer:
-  compute_nodes_config:
-    max_compute_nodes: 3
-standard:
-  compute_nodes_config:
-    max_compute_nodes: 9
+# The version of OpenShift on newly auto created data plane clusters.
+# If the value is empty (default), then the latest version is going to be used. 
+new_data_plane_openshift_version: ""
+# Whether to auto create a data plane  cluster 
+enable_dynamic_data_plane_scale_up: false
```

Comment:
Do you think this is specific enough? One thing is enabling the dynamic data plane scale up reconciling and another thing is the actual scaleup trigger

Reply:
This was mainly focused on the intent of scaling up (same for scale down) a cluster. 
Internally, setting this to false does not affect the evaluation but that's fine.

Reply:
If we are not being more specific on the name (which only affects the trigger) let's at least document it more thoroughly

Reply:
How would you like this documented?

Reply:
This controls the scale up trigger action

Reply:
Makes sense. I'll do it for the scale down as well.

Reply:
I've updated the documentation. Let me know what you think.